### DISCUSSION TITLE
formatted-output/println

### DISCUSSION THREAD

**Discussion by giscus** [sunface/rust-by-practice#522] [Book Comments]
# formatted-output/println

Learn Rust with Example, Exercise and real Practice, written with ❤️ by https://course.rs team

https://practice.course.rs/formatted-output/println.html

**Comment by DylanJinx** [comment]
done~

**Comment by AllenWu233** [comment]
Done.

**Comment by TDAkory** [comment]
mark finished

**Comment by Blyzes** [comment]
done
